pkg/archive/jar.go

```diff
@@ -10,6 +10,12 @@ import (
 	"strings"
 )
 
+var AllJarStabilizers []Stabilizer = []Stabilizer{
```

Comment:
nit: you should be able to omit the type on the var which will prevent declaration stutter.

Reply:
Removed it. My Java habbits seem to seep in here :sweat_smile: